_docs-sources/pipelines/architecture/index.md

```diff
@@ -0,0 +1,45 @@
+# Architecture
+
+## Overview
+
+There are three major components in the Gruntwork Pipelines architecture:
+
+1. The orchestrator
+2. The dispatcher
+3. The executor
+
+### The orchestrator
+
+The orchestrator identifies each [infrastructure change](../core-concepts/#infrastructure-change) in the Pull Request or git commit, classifies the type of infrastructure change it is (e.g. NewAwsAccount vs. OpenTfModuleUpdate) and determines the right [pipelines actions](../core-concepts/#pipelines-actions) to run based on that infrastructure change. For each infrastructure change, it calls the dispatcher.
```

Comment:
```suggestion
The orchestrator identifies each [infrastructure change](../core-concepts/#infrastructure-change) in the Pull Request or git commit, classifies the type of infrastructure change it is (e.g. NewAwsAccount vs. TerraformModuleUpdate) and determines the right [pipelines actions](../core-concepts/#pipelines-actions) to run based on that infrastructure change. For each infrastructure change, it calls the dispatcher.
```
We should avoid calling out open tf until it is available as a feature of pipelines

Reply:
Neither `NewAWSAccount` or `OpenTfModuleUpdate` are current infra change types in pipelines. You're likely looking for `ModuleUpdated`, `AccountRequested`, and `AccountAdded`

Reply:
Roger on OpenTF. Roger on infra change types; please feel free to correct as needed.